qcutilities.py

```diff
@@ -5,14 +5,20 @@
 import ROOT as r
```

Comment:
It's probably better to put this/these imports into the function that needs them

Reply:
moved